File: PooCC01CNetz/Semana02/Java/Ejercicios3/InvocarAtributos.java
public class InvocarAtributos {
    public static void main(String[] args) {
        // Creacion del objeto
        AlcanceAtributos aa = new AlcanceAtributos();
        aa.atributoPublico = "1111";
        aa.inicializaAP("2222");
        System.out.println("APublico    : " + aa.atributoPublico);
        System.out.println("APrivado    : " + aa.obtenerAP());

        AlcanceAtributos ab = new AlcanceAtributos("3333");
        System.out.println("APrivado    : " + ab.obtenerAP());
    }
}

File: PooCC01CNetz/Semana02/Python/Ejercicios3/InvocarAtributos.py
from AlcanceAtributos import *
def main():
    aa = AlcanceAtributos()
    aa.atributoPublico = "1111"
    aa.inicializaAP("2222")
    print("APublico     : " + aa.atributoPublico)
    print("APrivado     : " + aa.obtenerAP())

    ab = AlcanceAtributos("3333")
    print("APrivado     : " + ab.obtenerAP())

main()